### DISCUSSION TITLE
Requires Cloud Function

### DISCUSSION THREAD

**Discussion by mravani2** [AntlerVC/firetable#1087] [Support / Q&A] [answered]
I'm trying to use the Connector feature, but it's blocked:
![image](https://user-images.githubusercontent.com/122111577/213827685-66aec413-5a59-4cb5-a1bd-08548580ea1d.png)

I already did all the mentioned steps:
![image](https://user-images.githubusercontent.com/122111577/213827736-0ab03bc7-6d5f-4c97-b031-efc93e3c416f.png)

Yet, after confirming the last popup, I always see this, and I'm still not able to use the Connector feature:
![image](https://user-images.githubusercontent.com/122111577/213827802-e05aa1c0-520f-473c-bf30-b0ed243b0f53.png)

**Comment by bofeiw** [comment]
Can you check if your Firebase has `rowyRunUrl` in path `/_rowy_/settings`? If the setup is complete, you should see it.

https://console.firebase.google.com/u/0/project/[PROJECT_ID]/firestore/data/~2F_rowy_~2Fsettings

**Comment by BeeBombshell** [comment, marked as answer]
Hey @mravani2, I faced the same issue a while ago... 

## Steps to fix this:

1. Go to https://console.cloud.google.com/run for your project.
2. You'll be able to see two service instances, namely `rowy-hooks` and `rowy-backend`. Select both of them and click **delete**.
<img width="1113" alt="image" src="https://user-images.githubusercontent.com/53828745/214050529-b62c5482-2a1d-445a-8c42-6df4c62f552a.png">
3. Then, go back to your Rowy Project settings and try to re-deploy Rowy Run.

https://user-images.githubusercontent.com/53828745/214051524-f15c3125-64c2-4867-8a42-52cec2fe433c.mov

Hope this helps!

**Reply by mravani2** [reply]
Thank you so much, it's working now!
